frontend/packages/console-dynamic-plugin-sdk/src/api/dynamic-core-api.ts

```diff
@@ -196,39 +553,36 @@ export { useFlag } from '../utils/flags';
- * @param {ResourceYAMLEditorProps['initialResource']} initialResource - YAML/Object representing a resource to be shown by the editor.
- * This prop is used only during the inital render
+ * @param {ResourceYAMLEditorProps['initialResource']} initialResource - YAML/Object representing a resource to be shown by the editor. This prop is used only during the inital render
  * @param {ResourceYAMLEditorProps['header']} header - Add a header on top of the YAML editor
- * @param {ResourceYAMLEditorProps['onSave']} onSave - Callback for the Save button.
- * Passing it will override the default update performed on the resource by the editor
+ * @param {ResourceYAMLEditorProps['onSave']} onSave - Callback for the Save button. Passing it will override the default update performed on the resource by the editor
  */
 export const ResourceYAMLEditor: React.FC<ResourceYAMLEditorProps> = require('@console/internal/components/AsyncResourceYAMLEditor')
   .AsyncResourceYAMLEditor;
 
 /**
  * A component to show events related to a particular resource.
- *
  * @example
  * ```tsx
  * const [resource, loaded, loadError] = useK8sWatchResource(clusterResource);
  * return <ResourceEventStream resource={resource} />
  * ```
- * @param {ResourceEventStreamProps['resource']} - An object whose related events should be shown.
+ * @param {K8sResourceCommon} resource - An object whose related events should be shown.
  */
 export const ResourceEventStream: React.FC<ResourceEventStreamProps> = require('@console/internal/components/events')
   .WrappedResourceEventStream;
 
+
 /**
- * React hook to poll Prometheus for a single query.
- * @param options - Which is passed as a key-value map
- * @param options.query - Prometheus query string. If empty or undefined, polling is not started.
- * @param options.delay - polling delay interval (ms)
- * @param options.endpoint - one of the PrometheusEndpoint (label, query, range, rules, targets)
- * @param options.endTime - for QUERY_RANGE enpoint, end of the query range
- * @param options.samples - for QUERY_RANGE enpoint
- * @param options.timespan - for QUERY_RANGE enpoint
- * @param options.namespace - a search param to append
- * @param options.timeout - a search param to append
+ * Sets up a poll to Prometheus for a single query.
+ * @param {PrometheusEndpoint} props.endpoint - one of the PrometheusEndpoint (label, query, range, rules, targets)
+ * @param {string} [props.query] - (optional) Prometheus query string. If empty or undefined, polling is not started.
+ * @param {number} [props.delay] - (optional) polling delay interval (ms)
+ * @param {number} [props.endTime] - (optional) for QUERY_RANGE enpoint, end of the query range
+ * @param {number} [props.samples] - (optional) for QUERY_RANGE enpoint
+ * @param {number} [options.timespan] - (optional) for QUERY_RANGE enpoint
+ * @param {string} [options.namespace] - (optional) a search param to append
+ * @param {string} [options.timeout] - (optional) a search param to append
```

Comment:
@zherman0 Seems this part is needs to updated as well
![image](https://user-images.githubusercontent.com/78589509/174956767-65f2c2e2-7e26-45ba-a72d-abd12b2ddf2a.png)

Reply:
Again, the documentation is down correctly, just an issue with the markdown.

Reply:
@zherman0 I do not think we have a bug for this, can you log it?

Reply:
@andrewballantyne  -  A bug is opened for this issue: https://bugzilla.redhat.com/show_bug.cgi?id=2083154